File: backend/flight_booking_project/src/main/java/com/javaweb/flight_booking_project/dto/response/PlaneResponse.java
package com.javaweb.flight_booking_project.dto.response;


import lombok.Data;

import java.util.List;

@Data
public class PlaneResponse {
    private Long id;
    private int ecoClass;
    private int busClass;
    private String name;
    private String nameAirline;
}

File: mobile/lib/data/model/plane_response.dart
class PlaneResponse{
  int id;
  int ecoClass;
  int busClass;
  String name;

  PlaneResponse({required this.id, required this.name, required this.ecoClass, required this.busClass});

  factory PlaneResponse.fromJson(Map<String, dynamic> json) {
    return PlaneResponse(
      id: json['id'],
      name: json['name'],
      ecoClass: json['ecoClass'],
      busClass: json['busClass'],
    );
  }

  Map<String, dynamic> toJson() {
    return {
      'id': id,
      'name': name,
      'ecoClass': ecoClass,
      'busClass': busClass,
    };
  }
}